ChangeLog

```diff
@@ -1,3 +1,7 @@
+2024-08-15  Kevin Ushey  <kevinushey@gmail.com>
+
+	* src/raes.c: Calloc => R_Calloc; Free => R_Free
```

Comment:
I make that -> instead of => but all good.  Thanks for the PR ... and a ChangeLog entry!